src/main/scala/mesosphere/marathon/core/launcher/LauncherModule.scala

```diff
@@ -13,13 +13,13 @@ import mesosphere.marathon.metrics.Metrics
   * and actually launching the matched tasks.
   */
 class LauncherModule(
-    clock: Clock,
-    metrics: Metrics,
-    conf: MarathonConf,
-    taskCreationHandler: TaskCreationHandler,
-    marathonSchedulerDriverHolder: MarathonSchedulerDriverHolder,
-    offerMatcher: OfferMatcher,
-    pluginManager: PluginManager) {
+                      clock: Clock,
+                      metrics: Metrics,
+                      conf: MarathonConf,
+                      taskCreationHandler: InstanceCreationHandler,
+                      marathonSchedulerDriverHolder: MarathonSchedulerDriverHolder,
+                      offerMatcher: OfferMatcher,
+                      pluginManager: PluginManager) {
```

Comment:
probably not. There's probably an IntelliJ setting that I need to change. I
recently installed it and there's a bunch of things I'm still learning how
to configure.

On Wed, Sep 7, 2016 at 4:41 AM, Karsten Jeschkies <notifications@github.com>
wrote:

> In src/main/scala/mesosphere/marathon/core/launcher/LauncherModule.scala
> <https://github.com/mesosphere/marathon/pull/4313#discussion_r77781361>:
>
> > @@ -13,13 +13,13 @@ import mesosphere.marathon.metrics.Metrics
> >    * and actually launching the matched tasks.
> >    */
> >  class LauncherModule(
> > -    clock: Clock,
> > -    metrics: Metrics,
> > -    conf: MarathonConf,
> > -    taskCreationHandler: TaskCreationHandler,
> > -    marathonSchedulerDriverHolder: MarathonSchedulerDriverHolder,
> > -    offerMatcher: OfferMatcher,
> > -    pluginManager: PluginManager) {
> > +                      clock: Clock,
> > +                      metrics: Metrics,
> > +                      conf: MarathonConf,
> > +                      taskCreationHandler: InstanceCreationHandler,
> > +                      marathonSchedulerDriverHolder: MarathonSchedulerDriverHolder,
> > +                      offerMatcher: OfferMatcher,
> > +                      pluginManager: PluginManager) {
>
> Is this the proper indention? I'm just curious.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4313/files/af2ec891fd251891a78b95196c6ef4479487e23d#r77781361>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPVLMqegdDpqTvjXxhlx7BVXgI6o-iFks5qnnjNgaJpZM4J2U1J>
> .
>